source/_components/device_tracker.huawei_router.markdown

```diff
@@ -0,0 +1,36 @@
+---
+layout: page
+title: "Huawei Router"
+description: "Instructions how to integrate Huawei Routers into Home Assistant."
+date: 2017-07-16 01:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: bt.png
```

Comment:
Go to your fork, upload a new file with the github GUI and it will be added to the same PR.